js/renovation/utils/resolve_rtl.ts

```diff
@@ -0,0 +1,24 @@
+import { ConfigContextValue } from '../common/config_context';
+import { isDefined } from '../../core/utils/type';
+import globalConfig from '../../core/config';
+
+export function resolveRtlEnabled(rtlProp?: boolean, config?: ConfigContextValue):
+boolean | undefined {
+  if (rtlProp !== undefined) {
+    return rtlProp;
+  }
+  if (config?.rtlEnabled !== undefined) {
+    return config.rtlEnabled;
+  }
+  return globalConfig().rtlEnabled;
+}
+
+export function resolveRtlEnabledDefinition(rtlProp?: boolean, config?: ConfigContextValue):
+boolean {
+  const isPropDefined = isDefined(rtlProp);
+  const onlyGlobalDefined = isDefined(globalConfig().rtlEnabled)
+  && !isPropDefined && !isDefined(config?.rtlEnabled);
+  return (isPropDefined
+  && (rtlProp !== config?.rtlEnabled))
+  || onlyGlobalDefined;
+}
```

Comment:
These two functions look very similar. Can we use one of them inside another?

Reply:
No, we can't use one of them because they have different targets. resolveRtlEnabled - gets the total value from multiple sources (merge RTL properties), and resolveRtlEnabledDefinition - determines the source of enabling/disabling RTL (not value).